README.md

```diff
@@ -31,6 +31,12 @@ Using 3rd-party mirrors, for example
 ./kas-container build kas-iot2050-example.yml:kas/opt/mirror-example.yml
 ```
 
+Using swupdate with a A/B rootfs configuration, for example
+
+```shell
+./kas-container build kas-iot2050-example.yml:kas/opt/swupdate.yml
+```
+
```

Comment:
I suppose we will need "a bit" more explanation, what this is, what it does, what not (yet) etc.

Reply:
I will add some information to the end of the README.